types.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+type Params struct {
+	UpstreamName   string `json:"upstream"`
+	PagesDirectory string `json:"source"`
```

Comment:
fyi I'll send a follow-up pull request to change `upstream` to `remote` so that it matches some of our other plugins